tests/hazmat/primitives/test_ciphers.py

```diff
@@ -255,7 +255,7 @@ def test_update_into_buffer_too_small_gcm(self, backend):
     sys.platform not in {"linux", "darwin"}, reason="mmap required"
 )
 def test_update_auto_chunking():
-    large_data = mmap.mmap(-1, 2**29 + 2**20, prot=mmap.PROT_READ)
+    large_data = large_mmap(length=2**29 + 2**20)
```

Comment:
Regardless of the coverage bit, a PR that simply moved to a single utility function for these would be good.